app/assets/javascripts/client/common_list/entry_view.html

```diff
@@ -58,6 +58,7 @@ <h3 class='name'>{{entity.name}}
         <span class="stat">{{entity.participant_count}}</span>
         <span class="category">Completed</span>
         <span class="stat">{{entity.completed_responses}}</span>
+        <add-inventory-users-link data-inventory-id="entity.id"></add-inventory-users-link>
```

Comment:
We don't need the `data-` prefix on the parameter here.  Since we're replacing the custom HTML with the directive, there's no need to worry about using `data-`.